src/neptune/internal/backends/neptune_backend.py

```diff
@@ -305,6 +305,7 @@ def search_leaderboard_entries(
         query: Optional[NQLQuery] = None,
         columns: Optional[List[str]] = None,
         limit: Optional[int] = None,
+        sort_by: Optional[str] = None,
```

Comment:
I think this should be a non-optional. We have a default set to `sys/creation_time` and we have to somehow paginate over 10k parts so it's quite needed.